config/deploy.rb

```diff
@@ -26,7 +26,7 @@
 set :linked_files, fetch(:linked_files, []).push('config/database.yml', 'config/secrets.yml', 'config/faye.yml', 'config/shards.yml', '.env', '.ruby-version', '.ruby-gemset')
 
 # Default value for linked_dirs is []
-set :linked_dirs, fetch(:linked_dirs, []).push('log', 'tmp/pids', 'tmp/cache', 'tmp/sockets', 'vendor/bundle', 'public/system')
+set :linked_dirs, fetch(:linked_dirs, []).push('log', 'tmp/pids', 'tmp/cache', 'tmp/sockets', 'vendor/bundle', 'documents')
```

Comment:
Could this use ENV.fetch('DOCUMENTS_FOLDER'), otherwise it won't reflect what is set and could cause issues.